fuseprivate.c

```diff
@@ -79,7 +79,8 @@ int sqfs_usage(char *progname, bool fuse_usage, bool ll_usage) {
 	fprintf(stderr, "\n%s options:\n", progname);
 	fprintf(stderr, "    -o offset=N            offset N bytes into ARCHIVE to mount\n");
 	fprintf(stderr, "    -o subdir=PATH         mount subdirectory PATH of ARCHIVE\n");
-	fprintf(stderr, "    -o notify_pipe=PATH    named pipe that will receive 's' (success) or 'f' (failure) when the mountpoint is ready\n");
+	fprintf(stderr, "    -o notify_pipe=PATH    named pipe that will receive 's' (success)"
+			" or 'f' (failure) when the mountpoint is ready\n");
```

Comment:
Please indent the second line the way it is done on the `clone_fd` and `max_idle_threads` FUSE options, so the second line description starts at the same place as the first.

Reply:
Ah, you're referring to the actual output of the function and not the C code formatting.